dapps/marketplace/src/pages/listings/Listings.js

```diff
@@ -74,7 +74,7 @@ class Listings extends Component {
         <DocumentTitle pageTitle={<fbt desc="listings.title">Listings</fbt>} />
         <div className="container listings-container">
           {this.props.isMobile ? (
-            <Search className="search" placeholder />
+            <Search className="search" placeholder showDropdown />
```

Comment:
are there any instances of the Search component that don't need showDropdown?

Reply:
No, can remove that prop.